ocs_ci/helpers/dr_helpers.py

```
@@ -289,6 +296,19 @@ def failover(
     )
 
     drpc_obj.wait_for_phase(constants.STATUS_FAILEDOVER, timeout=360, sleep=10)
+
+    if skip_odf_cli_validation:
+        logger.info(
+            "Skipping ODF CLI validation for app-failover " "(primary cluster is down)"
```

Ruff (0.16.0)

[warning] Implicitly concatenated string literals on one line

Combine string literals

(ISC001)